### PR TITLE
Update readme for max input tokens and trim logits

### PR DESCRIPTION
**Pull request by puneeshkhanna**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by regisss — APPROVED** [review]
LGTM

**Comment by regisss** [comment]
CI failed but it's not related to this PR, merging
